modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/CopyFileEntryMVCRenderCommand.java

```diff
@@ -31,6 +48,54 @@
 public class CopyFileEntryMVCRenderCommand
 	extends BaseFileEntryMVCRenderCommand {
 
+	@Override
+	public String render(
+			RenderRequest renderRequest, RenderResponse renderResponse)
+		throws PortletException {
+
+		try {
+			FileEntry fileEntry = ActionUtil.getFileEntry(renderRequest);
+
+			if (!_dlSizeLimitConfigurationProvider.canCopy(
```

Comment:
change the if  to be the more readable
if (true){
return super...
}

(same comment than on the folder)